### PR TITLE
ENH: mucking around with the server test

### PR DESCRIPTION
**Pull request by klauer**
Mucking around classifies as "ENH", ... right?

(note: this is on top of @danielballan's `test-server` branch)

### CODE REVIEW

**Comment by danielballan** [comment]
Hahaha. I like the direction this is going in. When I have a dubious "ENH" I sometimes wish there were a standard abbreviation for "Pile of Technical Debt."

File: caproto/examples/curio_server.py [hunk 1]
```diff
@@ -111,19 +233,17 @@ def __init__(self, host, port, pvdb):
                     res = ca.VersionResponse(13)
                     responses.append(res)
                 if isinstance(command, ca.SearchRequest):
-                    known_pv = command.name.decode() in self.pvdb
+                    known_pv = command.name.decode('latin-1') in self.pvdb
```
Comment (danielballan) [inline caproto/examples/curio_server.py]: I didn't come across this. Does EPICS use latin-1 as opposed to UTF-8?

Comment (klauer) [inline caproto/examples/curio_server.py]: It uses... well, bytes with no specific encoding.  UTF-8 certainly wasn't around that time, and I think other than ASCII (limited to 7-bits) latin-1 is probably closest to what was intended.

That said, we could potentially make this a configurable option. I wonder what the other implementations do?

**Comment by klauer** [comment]
Closing in favor of #17

**Pull request closed without merging**